django/utils/log.py

```diff
@@ -162,7 +162,7 @@ def filter(self, record):
 class ServerFormatter(logging.Formatter):
     def __init__(self, *args, **kwargs):
         self.style = color_style()
-        super().__init__(*args, **kwargs)
+        super().__init__(*args, datefmt='%d/%b/%Y %H:%M:%S', **kwargs)
```

Comment:
Are we sure we don't get passed `datefmt` already? (As part of `args`/`kwargs`?)

I think we should have the signature here [match the super class](https://github.com/python/cpython/blob/5907e61a8d4da6d0f11bf1062d6d17484560a15e/Lib/logging/__init__.py#L555). Then provide a default if `datefmt is `None`: 

```
    def __init__(self, fmt=None, datefmt=None, style='%', validate=True)
        self.style = color_style()
        if datefmt is None: 
            datefmt = %d/%b/%Y %H:%M:%S'
        super().__init__(fmt, datefmt, style, validate) 
```

This way we don't crash no matter how we're instantiated, and folks can still customize by passing the parameter. 

(No doubt no one ever does but...) 

What do you think?

Reply:
Thanks for your suggestion. I've changed it a little bit because [`validate` was added in python3.8](https://docs.python.org/3/library/logging.html#logging.Formatter)

Reply:
Nice work on `validate` 👍